bin/alluxio-start.sh

```diff
@@ -12,18 +12,19 @@ BIN=$(cd "$( dirname "$0" )"; pwd)
 USAGE="Usage: alluxio-start.sh [-hNw] WHAT [MOPT] [-f]
 Where WHAT is one of:
   all MOPT\t\tStart master and all workers.
-  local\t\t\tStart a master and worker locally
-  master\t\tStart the master on this node
-  safe\t\t\tScript will run continuously and start the master if it's not running
-  worker MOPT\t\tStart a worker on this node
-  workers MOPT\t\tStart workers on worker nodes
-  restart_worker\tRestart a failed worker on this node
-  restart_workers\tRestart any failed workers on worker nodes
+  local [MOPT] \t\t\tStart a master and worker locally. SudoMount is assumed is MOPT is missing.
+  master\t\tStart the master on this node.
+  safe\t\t\tScript will run continuously and start the master if it's not running.
+  worker MOPT\t\tStart a worker on this node.
+  workers MOPT\t\tStart workers on worker nodes.
+  restart_worker\tRestart a failed worker on this node.
+  restart_workers\tRestart any failed workers on worker nodes.
 
 MOPT is one of:
   Mount\t\t\tMount the configured RamFS. Notice: this will format the existing RamFS.
   SudoMount\t\tMount the configured RamFS using sudo. Notice: this will format the existing RamFS.
-  NoMount\t\tDo not mount the configured RamFS
+  NoMount\t\tDo not mount the configured RamFS. Notice: Use NoMount and set ALLUXIO_RAM_FOLDER to
```

Comment:
I have a high-level question. Is it possible to make this work such that if NoMount is used, it will automatically use tmpFS instead of requiring the user to set the appropriate env variable? That could make this very user friendly.